driver/tke_driver.go

```diff
@@ -793,3 +790,99 @@ func (d *Driver) SetVersion(ctx context.Context, info *types.ClusterInfo, versio
 	logrus.Info("unimplemented")
 	return nil
 }
+
+// RemoveLegacyServiceAccount remove any old service accounts that the driver has created
+func (d *Driver) RemoveLegacyServiceAccount(ctx context.Context, info *types.ClusterInfo) error {
+	//func (d *Driver) RemoveLegacyServiceAccount(ctx context.Context, info *types.ClusterInfo) error {
+	clientSet, err := d.getClientSet(ctx, info)
+	if err != nil {
+		return err
+	}
+
+	return util.DeleteLegacyServiceAccountAndRoleBinding(clientSet)
+}
+
+// getClientSet returns cluster clientSet
+func (d *Driver) getClientSet(ctx context.Context, info *types.ClusterInfo) (kubernetes.Interface, error) {
```

Comment:
This func is duplicating logic that already exists in `PostCheck`. Please refactor the common logic out between them (I'd probably have `PostCheck` call `getClientSet`).

Reply:
refactored, thanks.